project/threads/migrations/0001_initial.py

```diff
@@ -0,0 +1,167 @@
+# Generated by Django 3.2.7 on 2021-09-24 16:33

```

Comment:
This migration contains the `Activity` and `Notification` models. Those models should be part of the `notifications` app instead of `threads`. Please delete this migration, restore the `Activity` and `Notification` models to their previous location, and then re-create the migration with only the thread-related models mentioned in #1032